packages/base/OutlinedInput/src/OutlinedInput/styles/styles.ts

```diff
@@ -0,0 +1,160 @@
+import type { FieldLayout } from '@toptal/picasso-form'
+import { twMerge } from '@toptal/picasso-tailwind-merge'
+
+import type { Props, Size } from '../OutlinedInput'
+
+const classesBySize: Record<Size, string> = {
+  small: 'text-xxs',
+  medium: 'text-md',
+  large: 'text-lg',
+}
+
+const getTextClasses = (size: Size | undefined) => {
+  if (!size) {
+    return ''
+  }
+
+  return `${classesBySize[size]} leading-4`
+}
+
+const getFontColorClass = ({
+  isDark,
+  disabled,
+  type,
+  inputPropsDisabled,
+}: {
+  isDark: boolean
+  disabled: boolean | undefined
+  type: string | undefined
+  inputPropsDisabled: boolean | undefined
+}) => {
+  if (isDark) {
+    return 'text-white'
+  }
+
+  if (type === 'time' && inputPropsDisabled) {
+    // It's a special case for TimePicker
+    return 'text-black'
+  }
+
+  if (disabled) {
+    return [
+      'text-gray-600',
+      // On Safari the text gets a bit lighter as if it had some transparency applied to it
+      // We need this webkit-specific property to achieve the exact font color
+      disabled ? '[&::-webkit-text-fill-color]:text-gray-600' : '',
+    ].join(' ')
```

Comment:
```suggestion
    ]
```

twJoin and twMerge accepts arrays, so we don't need to join these. It can even work with nested arrays